community/kernel/src/main/java/org/neo4j/kernel/api/proc/Procedure.java

```diff
@@ -36,7 +37,7 @@
 public interface Procedure
 {
     ProcedureSignature signature();
-    Stream<Object[]> apply( Context ctx, Object[] input ) throws ProcedureException;
+    RawIterator<Object[], ProcedureException> apply( Context ctx, Object[] input ) throws ProcedureException;
```

Comment:
Why apply returns <Object[], ProcedureException> instead of <T, ProcedureException> where T is type defined in procedure?